packages/WillowCreekApp/src/user-settings/Locations/index.js

```diff
@@ -74,6 +78,14 @@ class Location extends PureComponent {
     );
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  componentDidUnmount() {
+    /* We are forced to do manually switch back to `light-content` because this renders in a
+     * ModalView so it's the only way to override these styles. It is admitidly brittle. If you
+     * change anything related to status bar style be careful. */
+    StatusBar.setBarStyle('light-content');
```

Comment:
So the only problem with this, and I'm really not sure of a fix...but we open this screen from two places - one on the home screen where the status bar style is `light-content`, and one from the settings screen, wehre the status-bar style already is `dark-content`. I think this would effectively set the status bar back to light when you close it which would cause issue on the settings screen

Reply:
So I checked and it actually works ok! I was worried about that too. Feel free to test it.